code/modules/mob/living/carbon/alien/humanoid/alien_powers.dm

```diff
@@ -63,7 +63,7 @@ Doesn't work on other aliens/AI.*/
 	var/obj/machinery/atmospherics/components/unary/atmos_thing = locate() in user.loc
 	if(atmos_thing)
 		var/rusure = alert(user, "Laying eggs and shaping resin here would block access to [atmos_thing]. Do you want to continue?", "Blocking Atmospheric Component", "Yes", "No")
-		if(rusure != "No")
+		if(rusure == "No")
```

Comment:
!= "Yes"

Reply:
makes no difference whatsoever but i'll do it

Reply:
The difference is this way would make the player choose yes if they dc'd/crashed.